File: Projects/TransportathonHackathon.Clients/angular-ui/src/app/models/response-models/transport-requests/createdTransportRequestResponse.ts
export interface CreatedTransportRequestResponse {
  id: string;
  customerFirstName: string;
  customerLastName: string;
  transportTypeId: string;
  transportType: string;
  companyName: string;
  countryFrom: string;
  countryTo: string;
  cityFrom: string;
  cityTo: string;
  placeSize: string;
  createdDate: Date;
  approverByCompany: boolean;
  startDate: Date;
  finishDate: Date | null;
}

File: Projects/TransportathonHackathon.API/TransportathonHackathon.Application/Features/TransportRequests/Commands/Create/CreatedTransportRequestResponse.cs
namespace TransportathonHackathon.Application.Features.TransportRequests.Commands.Create
{
    public class CreatedTransportRequestResponse
    {
        public Guid Id { get; set; }
        public Guid TransportTypeId { get; set; }
        public string TransportType { get; set; }
        public bool IsPaid { get; set; }
        public string CustomerFirstName { get; set; }
        public string CustomerLastName { get; set; }
        public string CompanyName { get; set; }
        public string CountryFrom { get; set; }
        public string CountryTo { get; set; }
        public string CityFrom { get; set; }
        public string CityTo { get; set; }
        public bool ApprovedByCompany { get; set; }
        public bool IsFinished { get; set; }
        public string PlaceSize { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? FinishDate { get; set; }
    }
}
